### ISSUE TITLE
Edit the look of command signs via config?

### ISSUE THREAD

**Issue by TruDan** [alkarinv/BattleArena#272]
Hey

Can there be a config option for formatting the way the signs look, e.g. instead of

[GAMEMODE]
join
MapName
Open 0/0/12

Make it editable so we can make like

[Click to Join]
MapName
Open
0 / 12 Players

so like

signDisplay:
    '1': "[Click To Join]"
    '2': "%arena.name%"
    '3': "%arena.state%"
    '4': "%arena.players% / %arena.maxPlayers% Players"

Or something similar? This would be awesome if it could be added. I love how much configuration there is for everything else! Can literally make it do anything.

**Comment by creptor** [comment]
O.O found what you are looking for, it's an extra option:
//
signDisplayName: "&c[Click to Join]"
//

**Comment by TruDan** [comment]
But that won't update with the arena status and players etc

**Comment by creptor** [comment]
yea it will, it just changes the view line in the []

the format will be the same
[[game or display]]
join, leave
(name of the arena)
//automatic content(players, min players, max players))

**Comment by TruDan** [comment]
no, i mean it won't change the look of the other lines. I want to be able to change the positioning of the automatic content. Possibly with variables which get used in the config

**Comment by creptor** [comment]
O. Probably it won't be possible :( wait for the admin to respond
